docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -96,34 +96,34 @@ spec:
 EOF
 ----
 
-. Wait for Compute data plane service to be ready:
+. Wait for the Compute data plane services to be ready:
 +
 ----
-oc wait --for condition=Ready openstackdataplanedeployment/openstack-nova-compute-ffu --timeout=5m
+$ oc wait --for condition=Ready openstackdataplanedeployment/openstack-nova-compute-ffu --timeout=5m
```

Comment:
Should users replace openstackdataplanedeployment with the name of their CR? And should customers replace openstack-nova-compute-ffu with the name of their Compute data plane service?

Reply:
openstackdataplanedeployment is a hardcoded k8s API object name. While openstack-nova-compute-ffu is an example name, correct

Reply:
* openstackdataplanedeployment is hardcoded type name (CRD) in the openstack-operator. 
* openstack-nova-compute-ffu is user selected name but it is probably mentioned multiple places in this documents and we need that to be consistent, so if the name is changed we need to change it in every place. So while it can be changed I don't think we want to change it. It is a good enough name for the purpose.
* "nova" in the serviceOverried needs to match with the name of the service included in the DataPlaneNodeSet. So if the customer uses a custom service there e.g. called nova-custom, then here we need to change the serviceOverried to point to nova-custom instead as well.